File: Examples/Objective-C/KakaoAdSDK_DynamicLibraryExample/KakaoAdSDKExample/ViewController.m
//
//  ViewController.m
//  KakaoChipExample
//
//  Created by KAKAO on 2017. 12. 12..
//  Copyright © 2017년 KAKAO. All rights reserved.
//

#import "ViewController.h"
#import <KakaoAdSDK/KakaoAdSDK.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSString* tag = @"tag";
    
    // 가입
    [KakaoAdTracker sendCompleteRegisterEventWithTag:tag];
    
    // 장바구니 보기
    [KakaoAdTracker sendViewCartEventWithTag:tag];
    
    // 잠재고객
    [KakaoAdTracker sendParticipationEventWithTag:tag];
    
    // 가입 및 등록
    [KakaoAdTracker sendSignUpEventWithTag:tag];
    
    // 콘텐츠/상품 조회
    NSString* contentId = @"contentId";
    [KakaoAdTracker sendViewContentEventWithTag:tag contentId:contentId];
    
    // 검색
    NSString* searchString = @"robot";
    [KakaoAdTracker sendSearchEventWithTag:tag searchString:searchString];
    
    // 구매
    KakaoAdDetailProduct* product_1 = [[KakaoAdDetailProduct alloc] initWithName:@"pencil" quantity:5 price:400];
    KakaoAdDetailProduct* product_2 = [[KakaoAdDetailProduct alloc] initWithName:@"eraser" quantity:1 price:1000];
    KakaoAdDetailProduct* product_3 = [[KakaoAdDetailProduct alloc] initWithId:@"id-1" name:@"ruler" quantity:3 price:1500];
    
    NSInteger totalQuantity = 6;
    double totalPrice = 3000;
    NSString* currency = @"KRW";
    
    [KakaoAdTracker sendPurchaseEventWithTag:tag totalQuantity:totalQuantity totalPrice:totalPrice currency:currency products:@[product_1, product_2, product_3]];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Examples/Swift/KakaoAdSDKExample/KakaoAdSDKExample/ViewController.swift
//
//  ViewController.swift
//  KakaoChipExample
//
//  Created by KAKAO on 2017. 12. 12..
//  Copyright © 2017년 KAKAO. All rights reserved.
//

import UIKit
import KakaoAdSDK

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        let tag = "tag"
        
        // 가입
        KakaoAdTracker.sendCompleteRegisterEvent(tag: tag)
        
        // 장바구니 보기
        KakaoAdTracker.sendViewCartEvent(tag: tag)
        
        // 콘텐츠/상품 조회
        let contentId = "contentId"
        KakaoAdTracker.sendViewContentEvent(tag: tag, contentId:contentId)
        
        // 검색
        let searchString = "robot"
        KakaoAdTracker.sendSearchEvent(tag: tag, searchString: searchString)
        
        // 구매
        let product_1 = KakaoAdDetailProduct(name: "pencil", quantity: 5, price: 400)
        let product_2 = KakaoAdDetailProduct(name: "eraser", quantity: 1, price: 1000)
        let totalQuantity: Int = 6
        let totalPrice: Double = 3000
        let currency = "KRW"
        
        KakaoAdTracker.sendPurchaseEvent(tag: tag, totalQuantity: totalQuantity, totalPrice: totalPrice, currency:currency, products:[product_1, product_2])
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

